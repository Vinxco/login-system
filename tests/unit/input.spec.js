import { shallowMount } from "@vue/test-utils";
import InputSection from "@/components/Input.vue";

const factory = (values = {}) => {
  return shallowMount(InputSection, {
    data: function() {
      return {
        ...values
      };
    }
  });
};

describe("Input.vue", () => {
  it("El componente renderiza el mismo ‘snapshot’ que la última vez.", () => {
    const wrapper = factory();

    expect(wrapper).toMatchSnapshot();
  });

});
