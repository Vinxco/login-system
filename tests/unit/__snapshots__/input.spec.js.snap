// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input.vue El componente renderiza el mismo ‘snapshot’ que la última vez. 1`] = `<div class="form__section"><input id="" type=""> <label for=""></label></div>`;
